src/java/org/apache/cassandra/config/Config.java

```diff
@@ -522,6 +522,8 @@
      */
     public volatile int validation_preview_purge_head_start_in_sec = 60 * 60;
 
+    public boolean auth_cache_warming_enabled = true;
```

Comment:
nit: Did the original Jira description mention making this opt-in?

Reply:
Hm. Nope, should be optional. Changed to false and added to cassandra.yaml